Post/views.py

```diff
@@ -27,6 +28,18 @@ def CreateNewPost(request):
     return render(request, 'Post/createPost.html', {'form': form})
 
 
+class PostDeleteView(LoginRequiredMixin, UserPassesTestMixin, DeleteView):
+    model = Post
+    template_name = 'Post/deletePost.html'
+    success_url = '/postList/'
+
+    def test_func(self):
+        post = self.get_object()
+        if self.request.user == post.user:
+            return True
+        return False
```

Comment:
better to use `assert` instead of `if`

Reply:
@guy9050 I tried to use assert instead, but it didn't work that way

Reply:
@Shoval-Ovadia What was the problem